doc/nrf/protocols/wifi/promiscuous_operation.rst

```diff
@@ -0,0 +1,155 @@
+.. _ug_nrf70_developing_promiscuous_packet_reception:
+
+Raw IEEE 802.11 promiscuous packet reception
+############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF70 Series devices support Promiscuous mode.
+Promiscuous mode of operation is only supported in connected Station mode.
+In Promiscuous mode of operation, all received IEEE 802.11 packets that belong to the same BSS are sent to the receiving application by the nRF Wi-Fi driver.
+
+.. _ug_nrf70_developing_enabling_promiscuous_mode_feature:
+
+Enabling Promiscuous mode
+*************************
+
+To use the Promiscuous mode feature in your applications, you must enable the :kconfig:option:`CONFIG_NRF700X_PROMISC_DATA_RX` Kconfig option in the project configuration.
+
+.. _ug_nrf70_developing_promiscuous_mode_operation:
+
+Promiscuous mode operation
+**************************
+
+The nRF70 Series device supports raw 802.11 packet reception from a specific BSSID when Promiscuous mode is enabled for the nRF70 Wi-Fi driver.
+Promiscuous mode setting can be enabled when the nRF70 Series device operates in Station mode, however, Raw IEEE 802.11 packets in Promiscuousmode are only received when the nRF70 Series device connects to a specific BSSID and operates as a connected station.
+
+The ``net_eth_promisc_mode`` functional API can be used to enable or disable Promiscuous mode in the nRF Wi-Fi driver as required by the application.
+The ``NET_REQUEST_ETHERNET_SET_PROMISC_MODE`` network management API can also be used to enable or disable Promiscuous mode.
+However, it is recommended to use the functional API ``net_eth_promisc_mode`` for enabling or disabling Promiscuous mode in the nRF Wi-Fi driver.
+
+The following table provides the parameter details for ``net_eth_promisc_mode`` API which is used to enable/disable Promiscuous mode for raw packet reception.
+
+.. list-table:: Wi-Fi raw packet reception API parameter details
+   :header-rows: 1
+
+   * - API parameter
+     - description
+   * - iface
+     - Network interface on which Promiscuous mode is to be enabled
+   * - enable
+     - Boolean value to enable or disable Promiscuous mode.
+
+Since, Promiscuous packets are only received in connected Station mode, Promiscuous mode always operates in the channel in which the Station mode device is connected to the BSSID.
+
+You can refer to the relevant structure at:
```

Comment:
```suggestion
You can refer to the relevant structure in the following raw RX header file:
```
Then the text ` - for the raw RX header file` on line 48 can be removed.

Reply:
slightly modified suggested sentence as it is not just a raw RX header file. Kindly re-check